conbench/tests/api/test_runs.py

```diff
@@ -17,8 +17,8 @@
 }
 
 
-def _expected_entity(run, candidate_baseline_runs=None):
-    parent = run.commit.get_parent_commit()
+def _expected_entity(run: Run, candidate_baseline_runs=None):
```

Comment:
I suspect this small type hint would have prevented this bug in the first place 🙂